docs/pages/accounts/working-together.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Working Together
+---
+
+You can add other Expo users or members to your Personal Account. Members have access to all of the projects belonging to the account. The type of access depends on the given role.
```

Comment:
Until Orgs lands collaboration features are only available to Dev Services users.  I'd replace this line with something like "You can grant other users access to the projects belonging to your Personal Account with Expo Teams.  The type of access depends on the granted role.  You can sign up for Teams on any of your accounts at https://expo.io/dashboard/ACCOUNT/settings/services"

Reply:
Good point, I replaced it with the team reference! 👍